docs/dev/review.md

```
@@ -0,0 +1,114 @@
+
+### PR作成前
+```bash
+1. git push origin feature/xxx
+2. gh pr create ...
+3. /review owner/repo pr-number
+4. Blockers があれば修正
+5. 再度 /review で確認
+```
+
+### PRマージ判定
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)